pkgs/applications/version-management/bcompare/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv, qt511, libsForQt511, cairo, gdk_pixbuf, gnome2, bzip2, qt4, lib, fetchurl, autoPatchelfHook, coreutils }:
+
+stdenv.mkDerivation rec {
+  pname = "bcompare";
+  version = "4.2.10.23938";
+
+  src = fetchurl {
+    url = "https://www.scootersoftware.com/${pname}-${version}_amd64.deb";
+    sha256 = "5471d2bcb9494830bc706a940c8a598ecf7a1e56aa7e26c2694048b568d045a0";
+  };
+  sourceRoot = ".";
+
+  unpackPhase = ''
+    ar x $src
+    tar xfz data.tar.gz
+  '';
+
+  buildPhase = ":";
+
+  installPhase = with stdenv.lib; ''
+    mkdir -p $out/bin $out/lib $out/share
+    cp -R usr/share $out/
+    cp -R usr/lib $out/
+    cp -R usr/bin $out/
+
+    # Remove library that refuses to be autoPatchelf'ed
+    rm $out/lib/beyondcompare/ext/bcompare_ext_kde.amd64.so
+
+    find $out -name "bcompare" -exec sed -E -i "s:/usr/lib/beyondcompare:$out/lib/beyondcompare:g" {} \; -exec sed -E -i "s:/bin/bash:/usr/bin/env bash:g" {} \;
```

Comment:
Does it have to be this complicated with find?
Isn't the binary always going to be at `$out/bin/bcompare`?
Then you can use `substituteInPlace` for the lib and patchShebangs for bash.

Reply:
Nope, can remove replace the find, thx! /bin/bash is used inside the script as well, not only in shebang.